pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -3,10 +3,12 @@
 , glibmm, libsigcxx, lcms, boost, gettext, makeWrapper
 , gsl, python2, poppler, imagemagick, libwpg, librevenge
 , libvisio, libcdr, libexif, potrace, cmake, hicolor-icon-theme
+, scourSupport ? false
 }:
 
 let
-  python2Env = python2.withPackages(ps: with ps; [ numpy lxml ]);
+  python2Env = python2.withPackages(ps: with ps;
+    [ numpy lxml ] ++ stdenv.lib.optional scourSupport ps.scour);
```

Comment:
I think this can be
```
stdenv.lib.optional scourSupport scour
```
because there's `with ps;`

Reply:
Yeah right you are. I'll test it before pushing